iota/api.py

```diff
@@ -472,6 +691,22 @@ def get_trytes(self, hashes):
         Returns the raw transaction data (trytes) of one or more
         transactions.
 
+        :return:
+            ``dict`` with the following structure::
+
+                {
+                    'trytes': List[TransactionTrytes],
+                        List of transaction trytes for the given transaction
+                        hashes (in the same order as the parameters).
+                    'duration': int,
+                        Number of milliseconds it took to complete the request.
+                }
+
+        .. note::
+            If a node doesn't have the trytes for a given transaction hash in
+            its ledger, the value at the index of that transaction hash is either
+            ``null`` or a string of 9s.
```

Comment:
❔ Should these lines be indented more, so that they're part of the `:return:` block?

Reply:
Good point!